app/controllers/userAuth.js

```diff
@@ -0,0 +1,40 @@
+var jwt = require('jsonwebtoken');
+var mongoose = require('mongoose');
+var User = mongoose.model('User');
+var secret = 'ttv7yttvytftfftytftt';
+var moment = require('moment');
+
+module.exports.login = function(req, res) {
+  var body = req.body;
+  console.log(body);
```

Comment:
#### 🔴  Critical

Make sure once you are done debugging, all log statements `console.log()` are removed from your code base.